### ISSUE TITLE
Compiling error after install Laerdal.FFmpeg.Android with Xamarin

### ISSUE THREAD

**Issue by kikutano** [Laerdal/Laerdal.Xamarin.FFmpeg.Android#1]
**Describe the bug**
I installed Laerdal.FFmpeg.Android, but when I try to compile a lot of errors comes out like:

`The Java type `com.arthenica.mobileffmpeg.ExecuteCallback` is generated by more than one managed type. Please change the [Register] attribute so that the same Java type is not emitted.`

I really don't know how to fix this.

I'm using:
**Laerdal.FFmpeg.Video 4.4.27 on Core application
Laerdal.FFmpge.Video 4.4.28 on Android ( Target Framework Android 11.0 )**

**Comment by framinosona** [comment]
Hi @kikutano , have you tried to align the versions ? the same version on Core and Android project ?

**Comment by kikutano** [comment]
> Hi @kikutano , have you tried to align the versions ? the same version on Core and Android project ?

Yes, but for some reason I have a compatibility error when I try to Install the package on Android. So I just installed the plugin into the Core and it seems to work now.

**Comment by kikutano** [comment]
> Hi @kikutano , have you tried to align the versions ? the same version on Core and Android project ?

I can't find 4.4.27 version on Android, just 4.4.28. The core version is 4.4.27 and not 4.4.28.

Any idea? I can't compile on Android.
